### PR TITLE
update sfx-python to otel-python and fix otel-js endpoint

### PR DESCRIPTION
**Pull request by harnitsignalfx**

### CODE REVIEW

File: src/emailservice/requirements.txt [hunk 1]
```diff
@@ -11,14 +11,8 @@ grpcio==1.30.0            # via -r requirements.in, grpcio-health-checking, open
...
-opentelemetry-api==0.12b0  # via -r requirements.in, opentelemetry-exporter-zipkin, opentelemetry-instrumentation, opentelemetry-instrumentation-dbapi, opentelemetry-instrumentation-grpc, opentelemetry-instrumentation-requests, opentelemetry-instrumentation-sqlite3, opentelemetry-sdk
-opentelemetry-exporter-zipkin==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation-dbapi==0.12b0  # via opentelemetry-instrumentation-sqlite3
-opentelemetry-instrumentation-grpc==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation-requests==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation-sqlite3==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation==0.12b0  # via -r requirements.in, opentelemetry-instrumentation-dbapi, opentelemetry-instrumentation-requests, opentelemetry-instrumentation-sqlite3
-opentelemetry-sdk==0.12b0  # via -r requirements.in, opentelemetry-exporter-zipkin, opentelemetry-instrumentation-grpc
+splunk-opentelemetry
+opentelemetry-instrumentation-grpc==0.19b0  # via -r requirements.in
```
[the remark this answers is not in the document]
Comment (harnitsignalfx) [reply src/emailservice/requirements.txt:15]: Ah shoot. I should fix this, probably why things aren't looking good. A few are included by splunk-otel - 
opentelemetry-instrumentation-falcon>=0.15b0
opentelemetry-instrumentation-jinja2>=0.15b0
opentelemetry-instrumentation-requests>=0.15b0
opentelemetry-instrumentation-sqlite3>=0.15b0
opentelemetry-exporter-jaeger>=0.15b0
I'll fix the rest

File: src/recommendationservice/requirements.txt [hunk 2]
```diff
@@ -9,14 +9,8 @@ chardet==3.0.4            # via requests
...
 idna==2.10                # via requests
-opentelemetry-api==0.13b0  # via -r requirements.in, opentelemetry-exporter-zipkin, opentelemetry-instrumentation, opentelemetry-instrumentation-dbapi, opentelemetry-instrumentation-grpc, opentelemetry-instrumentation-requests, opentelemetry-instrumentation-sqlite3, opentelemetry-sdk
-opentelemetry-exporter-zipkin==0.13b0  # via -r requirements.in
-opentelemetry-instrumentation-dbapi==0.13b0  # via opentelemetry-instrumentation-sqlite3
-opentelemetry-instrumentation-grpc==0.13b0  # via -r requirements.in
-opentelemetry-instrumentation-requests==0.13b0  # via -r requirements.in
-opentelemetry-instrumentation-sqlite3==0.13b0  # via -r requirements.in
-opentelemetry-instrumentation==0.13b0  # via -r requirements.in, opentelemetry-instrumentation-dbapi, opentelemetry-instrumentation-requests, opentelemetry-instrumentation-sqlite3
-opentelemetry-sdk==0.13b0  # via -r requirements.in, opentelemetry-exporter-zipkin, opentelemetry-instrumentation-grpc
+opentelemetry-instrumentation-grpc==0.19b0  # via -r requirements.in
```
[the remark this answers is not in the document]
Comment (harnitsignalfx) [reply src/recommendationservice/requirements.txt:12]: Ack, fixing! Ty

File: src/emailservice/requirements.txt [hunk 3]
```diff
@@ -11,14 +11,8 @@ grpcio==1.30.0            # via -r requirements.in, grpcio-health-checking, open
...
 markupsafe==1.1.1         # via jinja2
-opentelemetry-api==0.12b0  # via -r requirements.in, opentelemetry-exporter-zipkin, opentelemetry-instrumentation, opentelemetry-instrumentation-dbapi, opentelemetry-instrumentation-grpc, opentelemetry-instrumentation-requests, opentelemetry-instrumentation-sqlite3, opentelemetry-sdk
-opentelemetry-exporter-zipkin==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation-dbapi==0.12b0  # via opentelemetry-instrumentation-sqlite3
-opentelemetry-instrumentation-grpc==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation-requests==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation-sqlite3==0.12b0  # via -r requirements.in
-opentelemetry-instrumentation==0.12b0  # via -r requirements.in, opentelemetry-instrumentation-dbapi, opentelemetry-instrumentation-requests, opentelemetry-instrumentation-sqlite3
-opentelemetry-sdk==0.12b0  # via -r requirements.in, opentelemetry-exporter-zipkin, opentelemetry-instrumentation-grpc
+splunk-opentelemetry
```
[the remark this answers is not in the document]
Comment (harnitsignalfx) [reply src/emailservice/requirements.txt:14]: ack

File: src/emailservice/requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (owais) [reply src/emailservice/requirements.txt:15]: Sounds good. Don't forget to use latest version (0.19b0) I think.

**Comment by harnitsignalfx** [comment]
Going to close this in favor of another. I screwed up my git history due to the /api/traces change.

**Pull request closed without merging**